check_runner/__init__.py

```diff
@@ -0,0 +1,153 @@
+"""
```

Comment:
A tad too verbose for a program description.

Maybe remove the project-management related aspects such as "not supported yet" ? They're mentioned in the README anyway